common/src/main/java/com/microsoft/identity/common/internal/fido/IFidoManager.kt

```diff
@@ -22,16 +22,15 @@
 // THE SOFTWARE.
 package com.microsoft.identity.common.internal.fido
 
-import android.webkit.WebView
-import com.microsoft.identity.common.internal.ui.webview.challengehandlers.IChallengeHandler
-
 /**
- * Abstract class that handles a FidoChallenge.
+ * Representation of a manager that handles interactions with a passkey provider (usually through an API).
  */
-abstract class AbstractFidoChallengeHandler
```

Comment:
This is an odd way that Git is depicting the change here, but I deleted the AbstractFIdoChallengeHandler class (don't need to shadow ChallengeHandler and we will be using the passkey challenge handler for security key support as well) and I'm creating a new IFidoManager interface,